linux_test/Makefile

```diff
@@ -3,7 +3,7 @@ all: sysv systemd upstart openrc clean
 
 # compile `go test` binary statically
 test:
-	@CGO_ENABLED=0 go test -installsuffix netgo -a -c ..
+	@CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go test -installsuffix netgo -a -c ..
```

Comment:
@kardianos can you give this a try. When running `make all` on darwin it was failing.

Reply:
It appears to build on my system, but I haven't tested it.

Reply:
I ran `linux_test/make all` for the changes I don't have a way to test windows.